docs/components/reactive-data.md

```diff
@@ -58,17 +58,34 @@ export class LoadingIndicator {
   }
 }
 ```
+
 In the example above, there are two `@Watch()` decorators.
 One decorates `watchPropHandler`, which will fire when the class member `activated` changes.
 The other decorates `watchStateHandler`, which will fire when the class member `busy` changes.
 
-When fired, the `@Watch()`'ed method will receive the old and new values of the prop/state.
+When fired, the decorated method will receive the old and new values of the prop/state.
 This is useful for validation or the handling of side effects.
 
 :::info
 The `@Watch()` decorator does not fire when a component initially loads.
 :::
 
+### Watching Native HTML Attributes
+
+Stencil's `@Watch()` decorator also allows you to watch native HTML attributes on the constructed host element. Simply
+include the attribute name as the argument to the decorator (this is case-sensitive):
+
+```tsx
+@Watch('aria-label')
+onAriaLabelChange(newVal: string, oldVal: string) {
+  console.log('Label changed:', newVal, oldVal);
+}
+```
+
+:::note
+Since native attributes are not `@Prop()` or `State()` members of the Stencil component, they will not trigger re-renders when changed.
```

Comment:
would it make sense for us to put a little recipe here for how you might enable that behavior (i.e. having something re-render when `aria-label` changes, for instance) ?

Reply:
Good idea! Added in 003ad90dbe1ce54eeb11be93c661cd059e036e1a